Description: Debian has a proper lib dependency system and doesn't need this check
Origin: vendor
Forwarded: not-needed
Author: Ryan Niebur <ryan@debian.org>

--- a/Glib.xs
+++ b/Glib.xs
@@ -475,6 +475,7 @@
 #endif /* GLIB_CHECK_VERSION (2, 24, 0) */
 	/* make sure that we're running/linked against a version at least as 
 	 * new as we built against, otherwise bad things will happen. */
+/*
 	if ((((int)glib_major_version) < GLIB_MAJOR_VERSION)
 	    ||
 	    (glib_major_version == GLIB_MAJOR_VERSION && 
@@ -488,6 +489,7 @@
 		      " old.  We'll continue, but expect problems!\n",
 		    GLIB_MAJOR_VERSION, GLIB_MINOR_VERSION, GLIB_MICRO_VERSION,
 		    glib_major_version, glib_minor_version, glib_micro_version);
+     */
 
 ##
 ## NOTE: in order to avoid overwriting the docs for the main Glib.pm, 
